packages/worker-api/src/integriteeWorker.ts

```diff
@@ -79,12 +103,21 @@ export class IntegriteeWorker extends Worker {
         from: String,
         to: String,
         amount: number,
+        note?: string,
```

Comment:
Internally, we will decide if it is the balance transfer call with note or without based on `note` being null or not.